src/sentry/tasks/data_export.py

```diff
@@ -44,19 +48,28 @@ def assemble_download(data_export_id):
                     file = File.objects.create(
                         name=file_name, type="export.csv", headers={"Content-Type": "text/csv"}
                     )
-                    file.putfile(tf)
+                    file.putfile(tf, logger=logger)
+                    logger.info("dataexport.end", extra={"id": data_export_id})
                     data_export.finalize_upload(file=file)
             except IntegrityError as error:
+                logger.error(
+                    u"dataexport.error: {}".format(error),
+                    extra={"query": data_export.payload, "org": data_export.organization_id},
```

Comment:
@leeandher Nit: I would use `organization_id` since that's what we typically use.